src/Core/Components/DataGrid/FluentDataGrid.razor.cs

```diff
@@ -243,10 +245,18 @@ public partial class FluentDataGrid<TGridItem> : FluentComponentBase, IHandleEve
     [Parameter]
     public bool AutoFit { get; set; }
 
+    [Parameter]
+    public DataGridRowSize RowSize { get; set; } = DataGridRowSize.Medium;
+
+    [Parameter]
+    public bool MultiLine { get; set; } = false;
```

Comment:
Can you add comments?

And why this property is on Table and not on each Column?

Reply:
With the way the grid is rendered now, every column's cell height  in a row needs to be set to 100% when MultiLine is true. We can't therefore set it on a column and it must be set on grid level.

Comments added